### PR TITLE
423 update class names

### PR DESCRIPTION
**Pull request by jwillmartin**
# PR Details
## Description
Changes class names within plugins that have recently been updated.
Added additional changes to default forwarding channel number to 183.

## Related Issue
https://github.com/usdot-fhwa-OPS/V2X-Hub/issues/423 

## Motivation and Context
Class names changed to reflect changes in plugin names/functionality.

## How Has This Been Tested?
Tested locally and tested generated docker image to ensure plugin functionality was preserved. 

## Types of changes

- [x] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp [hunk 1]
```diff
@@ -137,7 +137,7 @@
   * @param msg BSMMessage that is received
   * @routeable_message not used
   */
- void ODELoggerPlugin::HandleRealTimePublish(BsmMessage &msg,
+ void ODEForwardPlugin::HandleRealTimePublish(BsmMessage &msg,
```
Comment (rob-baily) [inline src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp:140]: @jwillmartin I know you did not change the code at line 145 below with the "new char" allocation but this seems like a fairly serious memory leak (as noted by Sonar) so do you think we can fix it with this PR?  You could do one of:

- Use a unique_ptr
- use stringstream
- Free it up at the end

File: src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp [hunk 2]
```diff
@@ -148,13 +148,14 @@
 
         //  check for schedule
         if(_freqCounter++%_scheduleFrequency == 0)
-                QueueKafkaMessage(kafka_producer, _BSMkafkaTopic, teststring);	
-	
+                QueueKafkaMessage(kafka_producer, _BSMkafkaTopic, teststring);
+		
+		delete teststring;
```
Comment (rob-baily) [inline src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp:153]: @jwillmartin This delete should be outside of this control block as the memory should be freed regardless of whether or not it is sent to Kafka.

Comment (jwillmartin) [reply src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp:154]: @rob-baily My bad, forgot to include the brackets on the if statement. Just pushed changes.

Comment (rob-baily) [reply src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp:154]: Ah I see that yours was actually fine but I do think that having the brackets helps make it clear..

File: src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp [hunk 3]
```diff
@@ -147,14 +147,16 @@
...
 
         //  check for schedule
-        if(_freqCounter++%_scheduleFrequency == 0)
-                QueueKafkaMessage(kafka_producer, _BSMkafkaTopic, teststring);	
-	
+        if(_freqCounter++%_scheduleFrequency == 0) {
+			QueueKafkaMessage(kafka_producer, _BSMkafkaTopic, teststring);
+		}
+		
+		delete teststring;
```
Comment (rob-baily) [inline src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp:154]: @jwillmartin Sorry to keep finding new things on this!  Sonar says "Memory allocated by 'new[]' should be deallocated by 'delete[]', not 'delete'" and claims segfaults can happen if the wrong delete is used.  You can check out their comments at https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=BUG&pullRequest=459&id=usdot-fhwa-ops_V2X-Hub&open=AYVZ04rg0fVLKmkUMLZQ

Comment (jwillmartin) [reply src/v2i-hub/ODEForwardPlugin/src/ODEForwardPlugin.cpp]: Ha, all good. Fixed!

**Review by rob-baily — APPROVED** [review]
The changes looks fine to me now.

**Pull request merged**